app/views/content/life-as-a-teacher/age-groups-and-specialisms/primary/_article.html.erb

```diff
@@ -69,7 +69,7 @@
     <p>Religious education, and relationships and health education are also compulsory for all state-funded schools.</p>
     <p>At the end of year 4, pupils will take a multiplication tables check to see if they have learnt the skills which will help them with increasingly challenging maths.</p>
     <p>At the end of KS2, pupils will sit national assessments in maths, science, English reading and writing. They'll then transition to secondary school. This is an exciting stage for pupils as they'll be ready to take on the new challenges of increasing independence and more in depth studies.</p>
-    <p>If teaching primary sounds like the career for you, find out more about <a href="/steps-to-become-a-teacher">how to become a teacher</a>.</p>
+    <%= render CallsToAction::ArrowLinkComponent.new(link_target: "/steps-to-become-a-teacher", link_text: "Find out more about how to become a teacher") %>
```

Comment:
```suggestion
    <%= render CallsToAction::ArrowLinkComponent.new(link_target: "/steps-to-become-a-teacher", link_text: "Find out how to become a teacher") %>
```

We haven't told them anything about **how** to become a teacher yet, so this doesn't need `more`?